### PR TITLE
Inflation Nomination for BL

### PR DESCRIPTION
**Pull request by brianlong**
#### Problem
BL is not currently listed as an inflation nominee.

#### Summary of Changes
Adds BL to the sdk/src/feature_set.rs script.

### CODE REVIEW

**Comment by mvines** [comment]
Hey @brianlong, PR looks good.  CI is dead right now, I'll help usher this along once we're back up again

**Review by mvines — APPROVED** [review]

File: sdk/src/feature_set.rs [hunk 1]
```diff
@@ -49,6 +49,17 @@ pub mod full_inflation {
...
     }
 
+    pub mod bl {
+        pub mod vote {
+            // The private key for this address is held by the Solana Foundation
+            solana_sdk::declare_id!("HRzoLj4jufnYEWosm9kWVgBVFdxAuqB1hu7vLckCuQHa");
+        }
+        pub mod enable {
+            // The private key for this address is held by BL
+            solana_sdk::declare_id!("BLxyQtJPzYZLHyj1p9n5QHUvbPoJt4TtRh7BXbG4M6rR.json");
```
Comment (mvines) [inline sdk/src/feature_set.rs:59]: Hey @brianlong, I removed the ".json" suffix here. Can you please confirm that the what remains here is in fact your public key:
```suggestion
            solana_sdk::declare_id!("BLxyQtJPzYZLHyj1p9n5QHUvbPoJt4TtRh7BXbG4M6rR");
```

Comment (brianlong) [reply sdk/src/feature_set.rs:59]: Gads! Sorry I missed that! Yes, this looks correct. Thanks for double-checking.

**Comment by mvines** [comment]
Unrelated CI failure

**Pull request merged**